testsuite/features/secondary/trad_ssh_tunnel.feature

```diff
@@ -4,12 +4,12 @@
 Feature: Register a traditional system to be managed via SSH push
 
   Scenario: Delete the traditional client for SSH reverse bootstrap
-    Given I am on the Systems overview page of this "sle_client"
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_client"
```

Comment:
Leave "sle_client" in this line, because at this point it is still one :-)

Reply:
OK; applied same fixes at `min_ssh_tunnel.feature`